admiral/pkg/clusters/registry.go

```diff
@@ -1026,18 +1026,28 @@ func addUpdateIstioResource(rc *RemoteController, m istio.Config, exist *istio.C
 		_, e = rc.IstioConfigStore.Create(m)
 
 	} else {
-		exist.Labels = m.Labels
-		exist.Spec = m.Spec
-		exist.Annotations = m.Annotations
-		verb = "Update"
-		_, e = rc.IstioConfigStore.Update(*exist)
+		if len(exist.Labels) > 0 {
+			if _, ok := exist.Labels["disable-update"]; ok {
```

Comment:
Can you please change something to indicate for admiral? Because `disable-update` is generic label. Like `disable-admiral-update` WDUT?